### ISSUE TITLE
Accessibility: No label is defined for the ‘Web Address(URL)’ & ‘Display as’ edit fields at ‘Insert link’ popup window.

### ISSUE THREAD

**Issue by shubhamsoni-ally** [microsoft/roosterjs#1884]
**Describe the bug**
No label is defined for the ‘Web Address(URL)’ & ‘Display as’ edit fields at ‘Insert link’ popup window.

**Pre-requisite**
Add the extension for Accessibility insights for Web [https://accessibilityinsights.io/](url)

**To Reproduce**
1. Open roosterjs demo website
2. Click on insert link
3. Run fastpass and observe the label issue

**Expected behavior**
Form element should have an implicit (wrapped) <label>
Form element should have an explicit <label>
aria-label attribute should exist 
aria-labelledby attribute should exist,
Element has should have title attribute
Element should have placeholder attribute

**Device Information**
 - OS: Windows 11
 - Browser: Edge
 - Version: 114.0.1823.43

**Comment by JiuqingSong** [comment]
Thanks for reporting. Would you like to submit a fix? @shubhamsoni-ally

**Comment by JiuqingSong** [comment]
Hi @shubhamsoni-ally , do you want to submit a fix? Or I'll close this issue for now since we don't plan to fix it from our side. Thanks
